cloud_process_node:
  ros__parameters:
    # 基础参数
    x_min: -0.5
    x_max: 0.5
    y_min: -0.5
    y_max: 0.5
    z_min: -0.5
    z_max: 0.5
    barrier_min_z: 0.4
    barrier_max_z: 0.5
    outlier_radius: 0.5
    outlier_min_neighbors: 5
    outlier_mean_k: 50
    outlier_std_dev: 1.0
    target_frame: "odom"
    lidar_frame: "livox_frame"
    base_frame: "base_link"
    